lib/fathead/redis-py/parse.py

```diff
@@ -0,0 +1,206 @@
+
+        for section in self.classes:
+            for sub_section in section:
+                if isinstance(sub_section, NavigableString):
+                	continue
+                if sub_section:
+                    if len(class_data) > len(class_desc):
+                        class_desc.append(sub_section)
+                    else:
+                        class_data.append(sub_section)
+
+        for section in zip(class_data, class_desc):
+            class_name, class_anchor = self.parse_name_and_anchor_from_data(section[0])
+            class_code = self.parse_code_from_data(section[0])
+            class_para = self.parse_paragraph_from_data(section[1])
+            data_elements = {
+                'name': class_name,
+                'anchor': class_anchor,
+                'paragraph': class_para,
+                'code': class_code
+            }
+            data.append(data_elements)
+
+            for sub_section in section[1].find_all('dl', {'class': 'method'}):
+                method_data, method_desc = list(), list()
+                for child in sub_section.children:
+                    if not child or isinstance(child, NavigableString):
+                        continue
+                    if len(method_data) > len(method_desc):
+                        method_desc.append(child)
+                    else:
+                        method_data.append(child)
+                for method in zip(method_data, method_desc):
+                    name, anchor = self.parse_name_and_anchor_from_data(method[0])
+                    para = self.parse_paragraph_from_data(method[1])
+                    code = self.parse_code_from_data(method[0])
+                    data_elements = {
+                        'name': name,
+                        'anchor': anchor,
+                        'paragraph': para,
+                        'code': code
+                    }
+                    data.append(data_elements)
+
+        self.parsed_data = data
+
+    def get_data(self):
+        """
+        Get the parsed data.
+        Returns:
+            self.parsed_data: Dict containing necessary data elements
+        """
+
+        return self.parsed_data
+
+class SoupDataOutput(object):
+    """
+    Object responsible for outputting data into the output.txt file
+    """
+    def __init__(self, data):
+        """
+        Initialize SoupDataOutput object
+        Args:
+            data: Dict containing the data elements
+        """
+        self.data = data
+
+    def create_file(self):
+        """
+        Iterate through the data and create the necessary output.txt file
+        """
+
+        with open('output.txt', 'w+') as output_file:
+            for data_element in self.data:
+                name = data_element.get('name').lower()
+                code = data_element.get('code')
+                paragraph = data_element.get('paragraph')
+                abstract = '<div class="prog__container">{}{}{}</div>'.format(
```

Comment:
@vidhan13j07 please use a `<section>` tag -- we found out `<div>` creates problems

Is there still a live example with `<div>`? I thought I fixed them all...